File: algorithm/src/main/java/baekjoon/p11320/Main.java
package baekjoon.p11320;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            System.out.println(solve(sc.nextInt(), sc.nextInt()));
        }
    }

    private static int solve(int a, int b) {
        return ((a * a) / (b * b));
    }

}

File: algorithm/src/main/java/baekjoon/p11320/Main.kt
package baekjoon.p11320

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        val inputs = readLine()!!.split(" ").map { it.toInt() }
        println(solve(inputs[0], inputs[1]))
    }
}

private fun solve(a: Int, b: Int): Int {
    return (a.times(a) / b.times(b))
}